ci-operator/step-registry/opendatahub-io/ci/s390x-build/dashboard/opendatahub-io-ci-s390x-build-dashboard-commands.sh

```diff
@@ -82,21 +82,9 @@ if [[ -n "${IMAGE_FLOATING_TAG-}" ]]; then
 fi
 
 echo "Image is $DESTINATION_IMAGE_REF"
-echo "commit hash is ${PULL_BASE_SHA:0:7}"
-echo "Entire commit hash is $PULL_BASE_SHA"
```

Comment:
You're removing some support for this variable -- but I see other references in the file... did you want to update the removal of all of them?

Reply:
I guess you are getting `PULL_BASE_SHA` from the environment and then you are setting it later in `ALL_VARS_STR` which is no longer needed there right?
Removing `PULL_BASE_SHA`is not gonna break the  postsubmit jobs right?